dm/loader/lightning.go

```diff
@@ -162,6 +166,11 @@ func (l *LightningLoader) Init(ctx context.Context) (err error) {
 		}
 		l.sqlMode = sqlModes
 	}
+
+	l.externalStore, err = exstorage.CreateExternalStore(ctx, l.cfg.LoaderConfig.Dir)
```

Comment:
will it cost a long time when `storage.New`? if so it should not be put in `Init`

Reply:
yes, we can call `storage.New` when it will be used. 
there may be error when we check dump files like [all_mode](https://github.com/pingcap/tiflow/blob/master/dm/tests/all_mode/run.sh#L489), because `storage.New` will create folder if path is not exist like [NewLocalStorage](https://github.com/pingcap/tidb/blob/master/br/pkg/storage/local.go#L129).
i think it will be better to change to call `storage.New` when it will be used.